http-client/src/test/groovy/io/micronaut/http/client/HostHeaderSpec.groovy

```diff
@@ -53,7 +53,7 @@ class HostHeaderSpec extends Specification {
         embeddedServer.close()
     }
 
-    @IgnoreIf({ System.getenv("TRAVIS")})
+    @IgnoreIf({ System.getProperty("os.name").contains("Linux") })
```

Comment:
In spock

`System.getProperty("os.name").contains("Linux") `

can be written with: 

`@IgnoreIf({ os.linux }) `